<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsValueRelationConfigDlgBase</class>
 <widget class="QWidget" name="QgsValueRelationConfigDlgBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Select layer, key column and value column</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Layer</string>
     </property>
     <property name="buddy">
      <cstring>mLayerName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsMapLayerComboBox" name="mLayerName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Key column</string>
     </property>
     <property name="buddy">
      <cstring>mKeyColumn</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QgsFieldComboBox" name="mKeyColumn"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Value column</string>
     </property>
     <property name="buddy">
      <cstring>mValueColumn</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QgsFieldComboBox" name="mValueColumn"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="mAllowNull">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Allow null value</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="mOrderByValue">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Order by value</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="mAllowMulti">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Allow multiple selections</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Filter expression</string>
     </property>
     <property name="buddy">
      <cstring>mValueColumn</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QToolButton" name="mEditExpression">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QTextEdit" name="mFilterExpression"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
